### DISCUSSION TITLE
如何與人交流？

### DISCUSSION THREAD

**Discussion by gigitilto** [1101-GS4719-C/Python-Community-GS4719-C#2233] [General]
我最近剛加入實驗室，一開始還與大家都不熟，所以能講的東西也比較多，但隨著時間過去，我感受到實驗室的學長姐逐漸不跟我說話，我知道可能是我看起來太過於陰沉，而且說話也不有趣，但我已經很努力想與他們交流了，我該怎麼辦，畢竟還要再相處兩年，我希望至少可以跟他們正常的對話，不然我可能待不下去這個實驗室了。

**Comment by Boying12** [comment]
我認為你可以試著主動尋找共同的話題，這樣可以打破與學長姐之間的隔閡。或許你可以從實驗室的研究內容著手，詢問他們對於某個實驗的看法或經驗，這樣不僅能讓他們感受到你的熱情，還能引發深入的討論。學長姐們在這個領域有著豐富的知識和經驗，他們會樂於分享，這樣的互動會讓你們的關係更進一步。
其次，參加一些實驗室的團隊活動或聚會也是一個好主意。這些非正式的場合能讓大家更輕鬆地互動，並建立更自然的關係。你可以主動提出一起喝咖啡、吃午餐或參與其他活動，這樣的方式有助於拉近你們之間的距離。

**Comment by Taoyi-Chiang** [comment]
要不要試試看團購一些東西，主訂發起訂飲料或午晚餐之類的，然後從這一家什麼東西好喝，那一家什麼便當好吃這樣開始。
慢慢的應該就會有越來越多可以一起聊天的內容吧？

**Comment by wcyeee** [comment]
感覺如果想要融入的話可以先多多了解想要融入的對象！因為感覺是為了以後要相處愉快所以想要和夥伴們變熟，所以如果不多做些什麼的話，可能因為單純興趣個性不合所以沒有機會變更熟悉。除了多多了解他們有更多共同話題之外我也覺得討論實驗的內容也是一個很棒的方法！

**Comment by forestdew507** [comment]
社交類問題似乎都頗看個案情況，但無論如何，我依然都會覺得在不妨礙他人的前提下自己舒服是最重要的 ! 或許不用刻意找話題，也許哪天話題就自己出現了:)

**Comment by qiu0109** [comment]
我覺得可以平常多問一點實驗上的問題，或是約學長姐實驗結束去吃點東西，但如果還是覺得無法融入的話也不用勉強，刻意找話聊反而會讓雙方都更不自在。

**Comment by Teddy111802518** [comment]
交流是門藝術，有人一身不社交有人卻朋友無數，你的每個用字都有意義，有藝術的講話可以包裝，我認為這是大家都需要的技術，像我最近也在學習，人是群聚動物，一身都會需要交流，好的交流習慣可以幫助生活
